cargo-miri/bin.rs

```diff
@@ -734,7 +734,7 @@ fn phase_rustc(mut args: env::Args, phase: RustcPhase) {
             // Ensure --emit argument for a check-only build is present.
             // We cannot use the usual helpers since we need to check specifically in `env.args`.
             if let Some(i) = env.args.iter().position(|arg| arg.starts_with("--emit=")) {
-                // For `no_run` tests, rustdoc passes a `--emit` flag; make sure it has the right shape.
+                // For `no_run` tests, rustdoc passes an `--emit` flag; make sure it has the right shape.
```

Comment:
I read this as "dash dash emit", but have no strong opinion

Reply:
I’ll remove the change here. I’ll rebase because the commit is small and this won’t impact reviewability.